jnigen/test/simple_package_test/java/com/github/dart_lang/jnigen/generics/MyStack.java

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2022, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
```

Comment:
And is it possible to also add test cases for some Java classes such as ArrayList and HashMap? :)